src/markdoc/partials/customPages/default-values.md

```diff
@@ -0,0 +1,121 @@
+{% /defaultvalue %}
+
+{% defaultvalue title="logger" defaultValue="pinoLogger for Node.js, null for browsers" %} 
+Determines the logger module responsible for handling SDK logs. 
+{% /defaultvalue %}
+
+{% defaultvalue title="logLevel" defaultValue="info" %} 
+Defines the log level. Available options: `debug`, `info`, `warn`, `log`, `error`.
+{% /defaultvalue %}
+
+{% defaultvalue title="enableLogging" defaultValue="true" %} 
+Enable or disable logging. Set to `false` to turn off all logging, even if a logger module is specified. 
+{% /defaultvalue %}
+
+## yagnaOptions
+
+{% defaultvalue title="apiKey" defaultValue="as read from 'process.env.YAGNA_APPKEY' for node.js, null for browser" %} 
+The `apiKey` facilitates access to the yagna REST API.
+{% /defaultvalue %}
+
+{% defaultvalue title="basePath" defaultValue="http://127.0.0.1:7465" %} 
+The base URL of the yagna REST API. 
+{% /defaultvalue %}
+
+{% defaultvalue title="maxTaskRetries" defaultValue="3" %} 
+Designates the maximum number of retry attempts for a job if it fails on the provider's side. 
+{% /defaultvalue %}
+
+{% defaultvalue title="activityPreparingTimeout" defaultValue="5 * 60 * 1000" %} 
+Timeout for activity preparation, which includes the creation and deployment of commands, in ms. 
+{% /defaultvalue %}
+
+## packagePackageOptions
+
+{% defaultvalue title="engine" defaultValue="vm" %} 
+Specifies the type of the payload. Options include: `vm`, `wasm`.
+{% /defaultvalue %}
+
+{% defaultvalue title="minMemGib" defaultValue="0.5" %} 
+Minimum required RAM size, denoted in gigabytes (GB). 
+{% /defaultvalue %}
+
+{% defaultvalue title="minStorageGib" defaultValue="2" %} 
+Minimum required storage size, denoted in gigabytes (GB).
+{% /defaultvalue %}
+
+{% defaultvalue title="minCpuThreads" defaultValue="1" %} 
+Minimum number of CPU threads required.
+{% /defaultvalue %}
+
+{% defaultvalue title="minCpuCores" defaultValue="1" %} 
+Minimum number of CPU cores required.
+{% /defaultvalue %}
+
+{% defaultvalue title="capabilities" defaultValue="[]" %} 
+List of capabilities that providers can offer. Refer to the manifest and demand documentation for more details. 
+{% /defaultvalue %}
+
+## MarketOptions
+
+{% defaultvalue title="debitNotesAcceptanceTimeout" defaultValue="30" %} 
+Sets the minimal time reserved for accepting debit notes. Proposals with shorter times will be rejected, measured in minutes. 
+{% /defaultvalue %}
+
+{% defaultvalue title="proposalFilter" defaultValue="acceptAllProposalFilter()" %} 
+This filter determines which proposals to accept, based on their compatibility with the demand.
+{% /defaultvalue %}
+
+## TaskServiceOptions
+
+{% defaultvalue title="maxParallelTasks" defaultValue="5" %} 
+Sets the maximum number of tasks that can be executed concurrently when the TaskExecutor processes tasks from a list (refer to `map()` and `forEach()` methods).
+{% /defaultvalue %}
+
+## PaymentOptions
+
+{% defaultvalue title="network" defaultValue="goerli" %} 
```

Comment:
I agree with your thoughts here. What should the text be replaced by instead? And do we have to re-order anything?

Reply:
this does not match the other templates... it would be best to create a `payment` field and describe it as consisting of two fields:
- network - description
- driver - description
and the default value is `{ network: "goerli", driver: "erc20" }`

Reply:
yes, this is a good solution, I approved it